src/model/CassandraClient.ts

```diff
@@ -0,0 +1,34 @@
+import { Client, mapping } from "cassandra-driver";
+
+const contactPoints = (process.env.CASSANDRA_HOSTS ?? "127.0.0.1").split(",");
```

Comment:
Ideally app specific environment variables should have no hardcoded defaults. Would rather add instructions in readme as to how to export variables for local/dev

Reply:
I am using NextJS's `.env` functionality: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables . All these values are defined in `.env.local` right now, and I can make a `.env` or `.env.production` for the final deployment. This hardcoded default is just to prevent nulls.